aten/src/ATen/native/TensorIterator.h

```diff
@@ -310,10 +313,15 @@ struct TensorIterator::Builder {
     iter_.resize_outputs_ = false;
   }
 
+  void check_internal_overlap(bool check_internal_overlap) {
```

Comment:
This is confusing to have `check_internal_overlap` doing completely different things based on arguments.